lua/nvim-lsp-installer/servers/omnisharp/init.lua

```diff
@@ -4,9 +4,30 @@ local path = require "nvim-lsp-installer.core.path"
 local functional = require "nvim-lsp-installer.core.functional"
 local github = require "nvim-lsp-installer.core.managers.github"
 local std = require "nvim-lsp-installer.core.managers.std"
+local Result = require "nvim-lsp-installer.core.result"
 
 local coalesce, when = functional.coalesce, functional.when
 
+local generate_cmd = function(root_dir)
+    if vim.fn.executable "mono" then
```

Comment:
This fn doesn't return a boolean, it returns an integer so you must check for == 0 or != 0. This currently forces mono even though it's not installed.